discord/responses.go

```diff
@@ -141,8 +147,9 @@ func lobbyMessage(g *GuildState) *discordgo.MessageEmbed {
 	//	Value:  "\u200B",
 	//	Inline: false,
 	//}
-	room, region := g.AmongUsData.GetRoomRegion()
-	gameInfoFields := lobbyMetaEmbedFields(&g.Tracking, room, region)
+	room, region, playerCount := g.AmongUsData.GetRoomRegion()
```

Comment:
Hm I like the compactness, but not sure I want the concept of the "room/region" values tied intrinsically to the number of players; partially in case future functionality would want to pull the two "sets" of values separately.

```
func (auData *AmongUsData) NumDetectedPlayers() int {
	auData.lock.RLock()
	defer auData.lock.RUnlock()
	return len(auData.playerData)
}
``` 
is what I came up with for this :)